frontend/views/reports.js

```diff
@@ -85,6 +133,7 @@ module.exports = Backbone.View.extend({
         this.$el.html(template(context));
         this.resultsView = new ReportTableView({
```

Comment:
initialize is like `__init__` and gets run when the view object is constructed, I think.  So constructing a new view object from inside there would probably lead to infinite recursion.  (http://backbonejs.org/#View-constructor)

I still don't think we need to construct a new object at all; these methods are all running in the context of a view object that already exists.